Bot/MessageHandlers/Commands/strike.js

```diff
@@ -0,0 +1,84 @@
+const ROLES = {
+    LUCKY : "902322660262957056"
+    STRIKEONE : "896026304237932565",
+    STRIKETWO : "906695092230832139",
+    BANNED : "614522888372092928",
+}
+
+function getMemberFromMention(mention, channel) {
+	if (mention.startsWith("<@") && mention.endsWith(">")) {
+		mention = mention.slice(2, -1);
+
+		if (mention.startsWith("!")) {
+            mention = mention.slice(1);
+			const foundMember = channel.members.find(
+                (member) => member.user.id === mention
+            );
+            return foundMember;
+		}
+
+        return undefined;
+	}
+}
+
+function addRoleById(msg, target, roleId) {
+    let role = msg.guild.roles.cache.find(r => r.id === roleId);
+    target.roles.add(role)
+        .then(() => {
+            console.log(`Given ${role} to <@${target.user.id}>`);
+        })
+        .catch(console.error);
+}
+
+function removeRoleById(msg, target, roleId) {
+    let role = msg.guild.roles.cache.find(r => r.id === roleId);
+    target.roles.remove(role)
+        .then(() => {
+            console.log(`Given ${role} to <@${target.user.id}>`);
+        })
+        .catch(console.error);
+}
+
+module.exports = {
+	name: "strike",
+	help: "Strike",
+	command: async function (msg, args) {
+        let body = args.join(" ");
+        let targetMember = getMemberFromMention(body, msg.channel);
+        if (!targetMember) {
+            msg.channel.send(`Could not find member: ${target}`);
+            return;
+        }
+
+        let targetId = targetMember.user.id;
+        let memberRoles = msg.member.roles.cache;
+
+        if (memberRoles.has(ROLES.LUCKY)) {
```

Comment:
Can a user ever get the "lucky" role from this command? Or would you have to assign it manually? In the latter case the naming doesn't make much sense

Reply:
Assigned manually, been adding it frequently enough so wanted to use it. Could possibly be used in other commands too to change the behaviour